src/TemplateEditor.js

```diff
@@ -212,6 +212,17 @@ class TemplateEditor extends React.Component {
 
     const invalid = (touched || submitFailed) && !valid && !showTokensDialog;
 
+    // DOMPurify reverses the order of attributes, so any supplied tags with attributes will ALWAYS
+    // have different output - so we just check for any changes DOMPurify might have made before using
+    // the string it produces.
+    let appliedValue = DOMPurify.sanitize(value, { ADD_TAGS: ['Barcode'], ADD_ATTR: ['target', 'rel'] });
+    if (value !== appliedValue) {
+      const removed = DOMPurify.removed.map((item) => item.attribute?.name || item.element?.outerHTML);
+      if (removed && removed.length === 0) {
+        appliedValue = value;
+      }
+    }
```

Comment:
Configuration wack-a-mole, unfortunately.. We need some solid use-cases.... Problematic thing drops in, ReactQuill wigs out... the script can be set to expose the changes that were removed so that those can be singled out in the config - but any change will mean a reset behavior for ReactQuill :(

Reply:
Rats. That means we're off to a good start (the editor is now usable when the value is clean) but since the editor is still broken if the value is problematic, then we have more work to do.